dandi/cli/cmd_validate.py

```diff
@@ -79,13 +79,20 @@ def validate_bids(
     default="none",
 )
 @click.option("--ignore", metavar="REGEX", help="Regex matching error IDs to ignore")
+@click.option(
+    "--min-severity",
```

Comment:
Also document this in `docs/source/cmdline/validate.rst`.

Reply:
ok, done.